egs/iwslt18st/st1/local/run_spm.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+# Copyright 2018 Kyoto University (Hirofumi Inaguma)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=-1       # start from -1 if you need to start from data download
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+
+# feature configuration
+do_delta=false
+
+# network archtecture
+# encoder related
+etype=vggblstm     # encoder architecture type
+elayers=3
+eunits=1024
+eprojs=1024
+subsample=1_2_2_1_1 # skip every n frame from input to nth layers
+# decoder related
+dlayers=2
+dunits=1024
+# attention related
+atype=add
+adim=1024
+aconv_chans=10
+aconv_filts=100
+
+# hybrid CTC/attention
+mtlalpha=0
+
+# minibatch related
+batchsize=32
+maxlen_in=800  # if input length  > maxlen_in, batchsize is automatically reduced
+maxlen_out=150 # if output length > maxlen_out, batchsize is automatically reduced
+
+# optimization related
+opt=adadelta
+epochs=20
+
+# rnnlm related
+lm_weight=0.3
+
+# decoding parameter
+beam_size=10
+penalty=0.0
+maxlenratio=0.0
+minlenratio=0.0
+ctc_weight=0
+recog_model=acc.best # set a model to be used for decoding: 'acc.best' or 'loss.best'
```

Comment:
this should be model.xxx
please check diff with other similar script, andcheck it carefully
also, please perform some small test before PR.

Reply:
Fixed it.